.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,276 @@
+name: Build LLDAP Bin and Dockerize
+
+on:
+  push:
+    branches: [ '*' ] #branch to build
+    tags: 'v*' #tag for specific release v1.0.2,or anytag, start with v
+  pull_request:
+    branches: [ 'main' ] #build test on main, not pushing
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+
+# In total 5 jobs, all of the jobs are containerized
+# ---
+
+# build-ui , create/compile the web
+## Use rustlang/rust:nighlty image
```

Comment:
Nit: nightly.

But actually, bigger nit: can we use stable? Or do we need nightly to cross-compile?

Reply:
If it passes, you'll just have to update the comments (here and below).

Reply:
Well technically pass, except the desc thing 👍🏼 
https://github.com/martadinata666/lldap/runs/7095765397